dags/utils/gcp.py

```diff
@@ -487,7 +487,9 @@ def bigquery_etl_query(
     """
     kwargs["task_id"] = kwargs.get("task_id", destination_table)
     kwargs["name"] = kwargs.get("name", kwargs["task_id"].replace("_", "-"))
-    sql_file_path = sql_file_path or "sql/{}/{}/query.sql".format(dataset_id, destination_table)
+    if not project_id:
+        project_id = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id
```

Comment:
Does this return shared-prod or does it return derived-datasets? I have trouble keeping track, since the Airflow cluster for Airflow is in derived-datasets.

Reply:
Right, it returns derived-datasets. We should default to shared-prod in this case though.